tests/test_api.py

```diff
@@ -68,6 +68,81 @@ def test_create(memory_store: Store) -> None:
         z = create(shape=(400, 100), chunks=(16, 16.5), store=store, overwrite=True)  # type: ignore [arg-type]
 
 
+LikeFuncName = Literal["zeros_like", "ones_like", "empty_like", "full_like", "open_like"]
+
+
+@pytest.mark.parametrize("func_name", get_args(LikeFuncName))
```

Comment:
Maybe there's a reason to structure it like this, but my usual preference is to use functions instead of names. So the actual params would be `[zarr.api.asynchronous.zeros_like, zarr.api.asynchronous.one_like, ...`].

Reply:
good catch, I don't remember why I used strings, but in https://github.com/zarr-developers/zarr-python/pull/2992/commits/19b1ce2650b69d3d879ef79e00cbc9a0ff3fe95f i switch to iterating over functions directly